langs/langs.json

```diff
@@ -1,4 +1,5 @@
 [
+  { "code": "ku", "name": "کوردی", "enName": "Kurdish" },
```

Comment:
Must use ckb for Central Kurdish.

Reply:
resolved :heavy_check_mark: with extra change
thanks for your response @roj1512